prow.sh

```diff
@@ -733,8 +746,55 @@ install_snapshot_controller() {
     sleep 10   
   done
 
+  SNAPSHOT_CONTROLLER_YAML="${CONTROLLER_DIR}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml"
+  if [[ ${REPO_DIR} == *"external-snapshotter"* ]]; then
+      NEW_TAG="csiprow"
```

Comment:
Can you add a comment where you got this logic from?

Reply:
This is here: https://github.com/kubernetes-csi/csi-release-tools/pull/113/files#diff-c2d96beb8cf73c62f7ae059aff96e381a8d73317df3ce356f36ba6b46338959dR1115

The image built from PR gets a "csiprow" tag.

Reply:
Added comments.

Reply:
Oh sorry, I was referring to the logic below where we're replacing the image tag

Reply:
That's from deploy-hostpath.sh: https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/util/deploy-hostpath.sh#L155

Reply:
Updated.